test/cpp/util/tls_test_utils.h

```diff
@@ -46,6 +46,25 @@ class SyncCertificateVerifier
   bool success_ = false;
 };
 
+// Exists to be an incorrect implementation that doesn't set a status
```

Comment:
optional: Consider just removing this comment, as the usage of this could change in the future.